pcicrawler/ocp_diag.py

```diff
@@ -0,0 +1,259 @@
+#!/usr/bin/env python3
```

Comment:
is this a script to be run as main module? if not, this shebang should not be here

Reply:
fine I'll drop the shebang